tests/cases/fourslash/findAllRefsOfConstructor.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: f.ts
+
+////class A {
+////    [|constructor|](s: string) {}
+////}
+////class B extends A { }
+////class C extends B {
+////    constructor() {
+////        [|super|]("");
+////    }
+////}
+////class D extends B { }
+////class E implements A { }
+////const a = new [|A|]("a");
+////const b = new [|B|]("b");
+////const c = new C();
+////const d = new [|D|]("d");
+////const e = new E();
+
+verify.noErrors();
+const [aCtr, cSuper, aNew, bNew, dNew] = test.ranges();
+verify.referenceGroups(aCtr, [
+    { definition: "class A", ranges: [aCtr, aNew] },
+    { definition: "class B", ranges: [cSuper, bNew]},
+    { definition: "class D", ranges: [dNew]}]);

```

Comment:
Can you please add tests for finding constructors for all other classes as well. Also tests for constructor invocation locations as well

Reply:
I tried to do this but find all references only understands that we are looking for a constructor reference if we call it on a constructor declaration. So if we call find all references on a constructor invocation like `new B()`, it will return references to the class `B`. I'm not sure if this is the expected behavior and don't know how to write those tests. Should I open an issue about this? @sandersn @sheetalkamat

Reply:
You might as well open an issue, although this seems like something that might have an existing issue.